README.md

```diff
@@ -25,25 +35,79 @@ jobs:
     name: Add issue to project
     runs-on: ubuntu-latest
     steps:
-      # Pointing to a branch name generally isn't the safest way to refer to an action,
-      # but this is how you can use this action now before we've begun creating releases.
-      # Another option would be to point to a full commit SHA.
       - uses: actions/add-to-project@main
         with:
           project-url: https://github.com/orgs/<orgName>/projects/<projectNumber>
           github-token: ${{ secrets.ADD_TO_PROJECT_PAT }}
           labeled: bug
 ```
 
+#### Further reading and additional resources
+
+- [Inputs](#inputs)
+- [Supported Events](#supported-events)
+- [How to point the action to a specific branch or commit sha](#how-to-point-the-action-to-a-specific-branch-or-commit-sha)
+- [Creating a PAT and adding it to your repository](creating-a-pat-and-adding-it-to-your-repository)
+- [Development](#development)
+- [Publish to a distribution branch](#publish-to-a-distribution-branch)
+
 ## Inputs
 
-- `project-url` is the URL of the GitHub Project to add issues to.
-- `github-token` is a [personal access
+- <a name="project-url">`project-url`</a> **(required)** is the URL of the GitHub Project to add issues to.
+- <a name="github-token">`github-token`</a> **(required)** is a [personal access
   token](https://github.com/settings/tokens/new) with the `repo`, `write:org` and
-  `read:org` scopes.
-- `labeled` is a comma-separated list of labels. For an issue to be added to the
-  project, it must have _one_ of the labels in the list. Omitting this key means
-  that all issues will be added.
+  `read:org` scopes.  
+  _See [Creating a PAT and adding it to your repository](creating-a-pat-and-adding-it-to-your-repository) for more details_
+- <a name="labeled">`labeled`</a> **(optional)** is a comma-separated list of labels used to filter applicable issues. When this key is provided, an issue must have _one_ of the labels in the list to be added to the project. Omitting this key means that any issue will be added.
+
+## Supported Events
+
+Currently this action supports the following [issue events](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#issues):
+
+- `opened`
+- `transferred`
```

Comment:
can we support `- labeled` here too?

Reply:
You're probably right I test it out tmw, I don't see why most of the issue events wouldn't work for this action, with the exception of things like `deleted` and maybe `locked`?

I took this list from a slack example thread but would be good to get a full list.

Reply:
I minimal list is probably ok. 

For instance we won't _remove_ and only intend to support labels for the initial release, so 

issue opened
issue transferred
issue labeled

should be good.   We can also land this and update later when we verify that!